pkgs/tools/networking/getmail6/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "getmail6";
+  version = "6.9";
+
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "09ii5z6nah21y6d49jfqrbqg6y6rvk764jg27yxlq4wlklhb4lmq";
+  };
+
+  doCheck = false;
```

Comment:
Why are tests disabled?

Reply:
Sorry, that was an oversight on my part. I've removed it, and re-pushed. Thanks for noticing it.

Reply:
Apparently it does not ship with tests, so I am keeping it.